### ISSUE TITLE
Please can you provide a requirements.txt

### ISSUE THREAD

**Issue by mccartnick** [aws-samples/generative-ai-amazon-bedrock-langchain-agent-example#1]
- I have run into an issue with the Claude bedrock model due to an outdated langchain lib version. 
- Of course this requires me to update the langchain version (to overcome this issue - https://github.com/langchain-ai/langchain/issues/11130) which is fixed in this commit (https://github.com/langchain-ai/langchain/commit/23065f54c047c71c9d71b724013f9652f9867930)
- When i updated the langchain version, I then run into a numpy version conflict. 
- In conclusion, it might be easier for a user to have a requirements.txt file that would allow them to directly modify and update their lambda layer.

**Comment by kyleblocksom** [comment]
Your error is a result of improper prompt template format, not outdated LangChain version. The Sample's prompt template has been updated to accommodate the new Claude Bedrock input requirements.
